packages/react-components/react-components/src/AccessibilityScenarios/Button.stories.tsx

```diff
@@ -71,38 +71,30 @@ export const MessengerButtonsAccessibilityScenario: React.FunctionComponent = ()
 
   return (
     <Scenario pageTitle="Messenger buttons">
-      <Button disabledFocusable={sendButtonDisabled} onClick={onSendButtonClick}>
-        Send
-      </Button>
-      <Button disabledFocusable={deleteButtonDisabled} onClick={onDeleteButtonClick}>
-        Delete
-      </Button>
       <Button ref={increaseFontButtonRef} disabled={increaseFontButtonDisabled} onClick={onIncreaseFontButtonClick}>
         Increase font size
       </Button>
       <Button ref={decreaseFontButtonRef} disabled={decreaseFontButtonDisabled} onClick={onDecreaseFontButtonClick}>
         Decrease font size
       </Button>
-      <div>
-        <textarea
-          name="message"
-          rows={3}
-          cols={50}
-          placeholder="Enter message here...."
-          aria-label="Message"
-          onChange={onMessageTextareaChange}
-          value={message}
-          style={messageStyle}
-        />
-      </div>
-      <p>
-        <span aria-live="polite">{statusText}</span>
-      </p>
+      <textarea
+        name="message"
+        rows={3}
+        cols={50}
+        placeholder="Enter message here...."
+        aria-label="Message"
+        onChange={onMessageTextareaChange}
+        value={message}
+        style={messageStyle}
+      />
+      <Button disabledFocusable={sendButtonDisabled} onClick={onSendButtonClick}>
```

Comment:
This is an interesting thing to test, but it feels pretty weird to have the send/delete buttons be disabled and focusable. If this is just used to gather feedback from users, it seems fine. But if we plan on having this on our public docs, we should probably update it so the send button at least is always enabled.

Reply:
It's not so easy to find a case when disabledFocusable would make sense, but you are right here... I have made the "Send" button enabled at all times. Do you think I could leave the "Delete" button disabledFocusable at least?

Reply:
How about making it `tabindex="-1"` when disabled and then that way if it becomes disabled because someone clicks it, keyboard focus doesn't shift, but it's still not in the tab order (unless enabled). I think that'd be a pretty good real-world sue case.

Reply:
That sounds like a good idea. So should I make the following changes?

* Leave the "Send" button always enabled.
* Make the "Delete" button disabled when activated, but set tabindex="-1" so that it remains in focus after becoming disabled.
* If there is no more steps for the "Decrease font size" or "Increase font size" buttons, make them disabledFocusable instead of disabled.
*

Reply:
So I've tried setting tabindex="-1" on the "Delete" button after clicking it, where clicking it also makes it disabled, but I cannot set focus programatically on the button afterwards, because it is not possible to set focus programatically on a disabled element. Therefore unfortunately what you suggested is not possible to achieve.